### ISSUE TITLE
Accessibility Feedback - Staging Review - Design System Action Links Documentation

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21416]
## Description
This issue will capture changes requested for the [Action Links](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/master/src/_components/action-links.md) and [Button](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/master/src/_components/buttons-action-links.md) documentation files.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

## Must
**Action Links:**
- TheAction Links image URL is broken. The visual explainers from Zepelin are really important to communicate intent.
- First point on the "When to consider something else" mentions Download. I'd like to remove this word from the bullet point. Downloading things like PDFs are still links because we're pointing to a resource with a unique URL that we're offloading instead of opening in a browser.
- The first bullet point in "Accessibility" should be removed. Action links are standard links that only respond to the `Enter` key.
- Please add a bullet point under "Accessibility" that Action Links must have an `href` attribute. They will not take keyboard focus without this attribute.

## Should
**Action Links:**
* Liquid template snippets should show syntax highlights. This can be achieved by adding a code fence around your snippets. Code fences can be created by typing three backticks and the name of the language to highlight.

  ```liquid
  <!-- Type ```liquid -->

  {% include_relative html/buttons-default.html %}

  <!-- Type ``` -->
  ```
## Consider
**Action Links:**
- Adding a bullet point to Action Links documentation under Accessibility for language attributes.

  Links that point to localized content in another language should have an `hreflang` attribute and a `lang` attribute in the following format:
  
  ```html
  <a
    className="va-action-link--blue"  
    href="#"
    hreflang="es"
    lang="es"
  >En Español</a>
  ```

**Buttons:**
* What do you think about renaming the [buttons-action-links.md](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/master/src/_components/buttons-action-links.md) file to something like **buttons.md** ? I wonder if teams reading the docs might be confused by "action links" being in the URL.

**Comment by CrystabelReiter** [comment]
**Must:**

- TheAction Links image URL is broken. The visual explainers from Zepelin are really important to communicate intent.
On the GH page it appears broken but I see it on this page: https://design.va.gov/components/action-links

- First point on the "When to consider something else" mentions Download. I'd like to remove this word from the bullet point. Downloading things like PDFs are still links because we're pointing to a resource with a unique URL that we're offloading instead of opening in a browser.
👍 Removed this

- The first bullet point in "Accessibility" should be removed. Action links are standard links that only respond to the Enter key
 👍 Removed this

- Please add a bullet point under "Accessibility" that Action Links must have an href attribute. They will not take keyboard focus without this attribute.
👍 Added this

For the "Should" and "Consider" feedback - I'll discuss with team and see how we can go forward 

For the Consider feedback around buttons. That's just a placeholder draft for buttons. I didn't want to take away the current buttons page and turn it into a draft. Once we've polished buttons-action-links.md we'll delete the current buttons.md and rename buttons-action-links.md  to buttons.md and publish the buttons docs with action link guidance

**Comment by 1Copenut** [comment]
This all sounds great, thank you @CrystabelReiter! I appreciated the Zepelin file showing all of the link states as part of the documentation artifacts. It's a strong visual reinforcement and check for teams as they start using action links.

**Comment by caw310** [comment]
@1Copenut , could you provide more information on the 'Should' section request. It's not clear to us what you mean by that and the purpose. Thanks!

**Comment by 1Copenut** [comment]
Sure thing @caw310 . The Liquid code snippets in the documentation would be easier to read if they were in code blocks. The comments were intended to explain what someone editing the Markdown file would need to type to get Liquid syntax highlights working.

**Comment by CrystabelReiter** [comment]
We've completed all the Must feedback, consider feedback and have discussed the should feedback. I'll close now
